src/jshint.js

```diff
@@ -3413,88 +3409,31 @@ var JSHINT = (function() {
     });
   }
 
-  var conststatement = stmt("const", function(context) {
+  function variableStatement(type, statement, context) {
```

Comment:
I'll put it back- it was just in the past it was a bit of a nightmare comparing let and const and var in the case of a bug - I can see how it has some advantages but it also makes the code prone to start to differ in ways they shouldn't.

Can I use the same function for let/const and seperate var or you'd prefer all 3 seperate?